frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/option_action.dart

```diff
@@ -41,7 +42,9 @@ enum OptionAction {
       case OptionAction.divider:
         return const FlowySvgData('editor/divider');
       case OptionAction.align:
-        return FlowySvgs.align_center_s;
+      // TODO: depth action icon required
+      case OptionAction.depth:
+        return FlowySvgs.m_aa_bulleted_list_s;
```

Comment:
How about using this one?
```suggestion
      case OptionAction.depth:
        return FlowySvgs.tag_s;
```